mirdata/giantsteps_key.py

```diff
@@ -256,10 +275,23 @@ def load_key(keys_path):
     if not os.path.exists(keys_path):
         raise IOError("keys_path {} does not exist".format(keys_path))
 
-    with open(keys_path) as f:
-        key = f.readline()
-
-    return key
+    with open(keys_path) as json_file:
+        new_annotations = json.load(json_file)
+    cardinality = (
+        [int(c) for c in new_annotations['cardinality']]
+        if new_annotations['cardinality'] is not None
+        else []
+    )
+    return {
+        'key': [global_key.split(' | ') for global_key in new_annotations['key']],
+        'confidence': new_annotations['confidence'],
+        'pitch-description': new_annotations['pitch-description'],
+        'pc-set': new_annotations['pc-set'],
+        'cardinality': cardinality,
+        'start_times': new_annotations['start_times'],
+        'end_times': new_annotations['end_times'],
+        'comments': new_annotations['comments'],
+    }
 
 
 def load_tempo(metadata_path):
```

Comment:
Maybe you don't need to create functions to load tempo if you can retrieve it from `metadata` in the `__init__` of Track. I think it's the same for `genres` and `artists`.
You would load the metadata with `utils.LargeData` and then: `self.genre = self._track_metadata['bpm']`. I think that would be easier and you wouldn't need to create all these catched properties and all these methods. 
Am I correct?